File: algo-fibonacci.js
function fibonacci (num) {
    
    let num1 = 0;
    let num2 = 1; 
    let fib;
    if (num<2){
        console.log(num);
    }
    else{
      for (let i = 1; i < num; i++){
        fib = num1 + num2;
        num1 = num2;
        num2 = fib;
      }
      console.log(fib);
    }
};

fibonacci(7);

File: algo-fibonacci.py
def fibonacci(num):
    
    num1 = 0
    num2 = 1
    if (num<2):
        print(num)

    else:
        for i in range(num):
            fib = num1 + num2
            num1 = num2
            num2 = fib
            i = i + 1

        print(fib)

fibonacci(5)